tests/js/client/shell/shell-index.js

```diff
@@ -112,9 +113,49 @@ function IndexSuite() {
       }
       assertTrue(seenProgress, "Never saw progress being reported!");
     },
+    
+    testIndexProgressIsNotReported : function () {
+      internal.debugSetFailAt("fillIndex::pause");
+      let idxdesc = { name: "progress", type: "persistent", fields: ["name"], inBackground: true};
+
+      let job = arango.POST_RAW(`/_api/index?collection=${cn}`, idxdesc,
+                                {"x-arango-async": "store"}).headers["x-arango-async-id"];
+      let count = 0;
+      // check for 5 seconds that the in-progress index does not appear in the result.
+      // using 5 elapsed seconds as the test's stop condition is not a guarantee that
+      // the in-flight index won't appear later, but we do not have a better stop
+      // condition for the test.
```

Comment:
Once we have machines which are fast enough to create the index within 5 seconds, this test will fail just because of this.

Reply:
I think we must do this differently: Wait for a long time (even for ASAN runs) until the index appears. And fail the test if - when it appears - it shows with either `isBuilding:true` or with `progress` being set.

Reply:
Wrong, I forgot the failure point. Please ignore this.